app/models/point_request.rb

```diff
@@ -8,4 +8,11 @@ class PointRequest < ActiveRecord::Base
 
   belongs_to :evaluation, foreign_key: :ertekeles_id
   belongs_to :user, foreign_key: :usr_id
+  has_many :point_details
+
+  def recalculate!
+    self.point = point_details.sum(:point)
+    save
```

Comment:
We have to use `save!` method, if the method name contains the exclamation mark.
Or deliberately do we not use the validation?

Reply:
The save method also does the correct validations. The only difference with the `save!` is that the latter throws an exception on a failed save.
And the `recalculate!` method name contains an exclamation mark because it [modifies the state of the object](https://github.com/bbatsov/ruby-style-guide#naming).

Still, it's a good idea to use `save!` because we don't want silent failures, so I changed it :smile: